gensim/models/word2vec.py

```diff
@@ -1272,6 +1299,17 @@ def _load_specials(self, *args, **kwargs):
             self.wv = wv
         super(Word2Vec, self)._load_specials(*args, **kwargs)
 
+    @classmethod
+    def load_word2vec_format(cls, fname, fvocab=None, binary=False, encoding='utf8', unicode_errors='strict',
```

Comment:
This now makes the `call_on_class_only` reference in `__init__()` superfluous/wrong.